pkgs/development/tools/spring-boot/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchzip, jdk, makeWrapper, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "spring-boot";
+  version = "2.1.8";
+
+  src = fetchzip {
+    url = "https://repo.spring.io/release/org/springframework/boot/${pname}-cli/${version}.RELEASE/${pname}-cli-${version}.RELEASE-bin.zip";
+    sha256 = "0shkznvv8abb365pgzfp265ihmf17mqigzzimdkzyp9jrh9kgiqx";
+  };
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  installPhase = ''
+    rm bin/spring.bat
```

Comment:
```suggestion
    runHook preInstall
    rm bin/spring.bat
```

Reply:
What features do preInstall and postInstall provide that is needed in this build? I have a few other similar packages that do not run these either.

Reply:
I added these changes.